hack/generate/csv.sh

```diff
@@ -49,14 +49,12 @@ image "3scale-kourier-control" "${registry}/knative-v$(metadata.get dependencies
 image "eventing-controller__eventing-controller"    "${eventing}-controller"
 image "sugar-controller__controller"                "${eventing}-sugar-controller"
 image "eventing-webhook__eventing-webhook"          "${eventing}-webhook"
-image "storage-version-migration-eventing__migrate" "${eventing}-storage-version-migration"
 image "mt-broker-controller__mt-broker-controller"  "${eventing}-mtchannel-broker"
 image "mt-broker-filter__filter"                    "${eventing}-mtbroker-filter"
 image "mt-broker-ingress__ingress"                  "${eventing}-mtbroker-ingress"
 image "imc-controller__controller"                  "${eventing}-channel-controller"
 image "imc-dispatcher__dispatcher"                  "${eventing}-channel-dispatcher"
 
-image "v0.17.0-pingsource-cleanup-eventing-$(metadata.get dependencies.eventing)__pingsource" "${eventing}-pingsource-cleanup"
 image "PING_IMAGE"           "${eventing}-ping"
 image "MT_PING_IMAGE"        "${eventing}-mtping"
```

Comment:
The image is no longer stored as an the `MT_PING_IMAGE` env-var on the `eventing-controller`


It is now its own deployment, like:
https://github.com/openshift-knative/operator/blob/release-0.18/cmd/operator/kodata/knative-eventing/0.18.0/3-eventing-core.yaml#L653-L675


I think it needs to be like:
```
image "pingsource-mt-adapter__dispatcher"                  "${eventing}-mtping"
```

and I'd than list it on line 57?

Reply:
He, thanks! I was just startring to look into that.